client/src/app/+videos/+video-edit/shared/thumbnail-manager/thumbnail-manager.component.html

```diff
@@ -0,0 +1,29 @@
+<div class="root">
+    <div *ngIf="!selectingFromVideo" class="preview-container">
+  
+      <img *ngIf="imageSrc" [ngStyle]="{ width: previewWidth, height: previewHeight }" [src]="imageSrc" class="preview"
+        alt="Preview" i18n-alt />
+      <div *ngIf="!imageSrc" [ngStyle]="{ width: previewWidth, height: previewHeight }" class="preview no-image"></div>
+  
+    </div>
+  
+    <div id="embedContainer" class="" style="position: relative;"></div>
+  
+    <div class="inputs">
+      <my-reactive-file *ngIf="!selectingFromVideo" inputName="uploadNewThumbnail" inputLabel="Upload image"
```

Comment:
Can you try to use `@if`/`@else` to it's clearer to read? https://blog.angular-university.io/angular-if/